docs/ug/cli.md

```diff
@@ -261,3 +261,40 @@ Cannot be used with any other flags. This flag takes precedence over all other f
   Default: `./reposense-report`
 * Alias: `-v`
 * Example:`--view` or `-v`
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### `--analyze-authorship`, `-A`
+
+**`--analyze-authorship`**: Performs further analysis to distinguish between partial and full credit attribution for
+lines of code assigned to the author.
+
+* Default: this feature is turned ***off*** by default and the author will receive full credit for all lines of code.
```

Comment:
Can you update the docs based on your recent change? Maybe you could include the rational behind it too (the line should be _at least_ partial credit) in a Markbind tooltip. 
```suggestion
* Default: this feature is turned ***off*** by default and the author will receive partial credit for all lines of code.
```